pages/common/flutter.md

```diff
@@ -3,9 +3,9 @@
 > Google's free, open source, and cross-platform mobile app SDK.
 > More information: <https://github.com/flutter/flutter/wiki/The-flutter-tool>.
 
-- Display help about a specific command:
+- Create a flutter project in current directory:
```

Comment:
`In the current directory` or `in a directory of the same name`?

Reply:
The command is  `flutter create {{directory_path}}`, so i think `Create a flutter project in specific directory` is better.